app/controllers/conclusion_final_reviews_controller.rb

```diff
@@ -383,6 +388,24 @@ def set_conclusion_final_review
       ).find(params[:id])
     end
 
+    def new_conclusion_final_review_params
```

Comment:
¿Este sigue haciendo falta? Porque por como está ahora debería funcionar como antes con el de abajo.

Reply:
si uso el otro franco no funciona porque toma la referencia a los anexos ya existentes que son del draft. Entonces cuando voy a hacer el new dice que no encuentra el anexo con id para ese conclusionFinalReview.

Reply:
`Couldn't find Annex with ID=17 for ConclusionFinalReview with ID=`

Reply:
Ahh claro, pero ahora como los "copiamos" ni deberían venir como params, de hecho debería quedar el "original" solamente. Si en algún momento dejamos que editen ahí si.

Reply:
Ahhh claro, porque yo le habia agregado al otro lo de los anexos. Olvidate me equivoque yo, el otro queda como estaba antes sin los anexos y este lo vuelo. Totalmente